core/src/main/java/org/infinispan/distribution/impl/L1ManagerImpl.java

```diff
@@ -163,15 +165,13 @@ private Collection<Address> buildInvalidationAddressList(Collection<Object> keys
       return addresses;
    }
 
-   private boolean isUseMulticast(int nodes) {
+   private boolean isUseMulticast(int invalidationNodes, int clusterSize) {
```

Comment:
I think we can't change it like this for the `threshold` value. We should keep the same verification as before, but only add the `thresholdRatio` validation. I see you updated the default value for `threshold`, but if a user was explicitly setting the value to `0`, it wouldn't multicast anymore.